website/docs/r/bastion_host.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_bastion_host"
+sidebar_current: "docs-azurerm-resource-bastion-host-x"
+description: |-
+  Manages a Bastion Host Instance.
+
+---
+
+# azurerm_bastion_host
+
+Manages a Bastion Host Instance.
+
+## Example Usage
+
+This example deploys an Azure Bastion Host Instance to a target virtual network.
+
+```hcl
+resource "azurerm_resource_group" "test" {
```

Comment:
COuld we use example for all resource names here?
```suggestion
resource "azurerm_resource_group" "example" {
```

Reply:
Changed to example for next push